src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -5,18 +5,15 @@ import {generateRandomId} from 'stream-chat-react-native-core';
 import {launchImageLibrary} from 'react-native-image-picker';
 import {openComposer} from 'react-native-email-link';
 import {useNavigation} from '@react-navigation/core';
-import {v4 as uuid} from 'uuid';
 
 import AnonymousMessageRepo from '../../../service/repo/anonymousMessageRepo';
-import ChannelList from '../../../database/schema/ChannelListSchema';
-import ChannelListMemberSchema from '../../../database/schema/ChannelListMemberSchema';
-import UserSchema from '../../../database/schema/UserSchema';
-import useChatUtilsHook from '../../../hooks/core/chat/useChatUtilsHook';
-import useLocalDatabaseHook from '../../../database/hooks/useLocalDatabaseHook';
+import useCreateChat from '../../../hooks/screen/useCreateChat';
+import TokenStorage, {ITokenEnum} from '../../../utils/storage/custom/tokenStorage';
+import {ANONYMOUS_USER} from '../../../hooks/core/constant';
 import {Context} from '../../../context';
 import {checkUserBlock} from '../../../service/profile';
 import {getChatName} from '../../../utils/string/StringUtils';
-import {getOrCreateAnonymousChannel} from '../../../service/chat';
+import {isContainUrl} from '../../../utils/Utils';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been modified. It seems like the `useCreateChat` hook is being used instead of the `useChatUtilsHook` and `useLocalDatabaseHook`. This could potentially change the behavior of the code if the new hook does not perform the same operations as the old ones. Please ensure that the new hook `useCreateChat` has all the necessary functionalities required by this file.

```diff
- import useChatUtilsHook from '../../../hooks/core/chat/useChatUtilsHook';
- import useLocalDatabaseHook from '../../../database/hooks/useLocalDatabaseHook';
+ import useCreateChat from '../../../hooks/screen/useCreateChat';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->